be/src/connector/lake_connector.cpp

```diff
@@ -311,9 +310,11 @@ Status LakeDataSource::get_next(RuntimeState* state, ChunkPtr* chunk) {
 }
 
 Status LakeDataSource::get_tablet(const TInternalScanRange& scan_range) {
-    _version = strtoul(scan_range.version.c_str(), nullptr, 10);
-    ASSIGN_OR_RETURN(auto tablet, ExecEnv::GetInstance()->lake_tablet_manager()->get_tablet(scan_range.tablet_id));
-    ASSIGN_OR_RETURN(_tablet_schema, tablet.get_schema());
+    int64_t tablet_id = scan_range.tablet_id;
+    int64_t version = strtoul(scan_range.version.c_str(), nullptr, 10);
+    ASSIGN_OR_RETURN(_tablet, ExecEnv::GetInstance()->lake_tablet_manager()->get_tablet(tablet_id, version));
+    CHECK(_tablet.valid());
```

Comment:
maybe DCHECK instead of check? avoid crash BE in release mode?

Reply:
Change CHECK to DCHECK will not prevent process crash in release mode, but just let it crash in the nextl ine..

Reply:
I removed this assertition